src/sentry/runner/commands/run.py

```diff
@@ -253,6 +253,61 @@ def taskworker(rpc_host: str, max_task_count: int, **options: Any) -> None:
         raise SystemExit(exitcode)
 
 
+@run.command()
+@log_options()
+@configuration
+@click.option(
+    "--repeat",
+    type=int,
+    help="Number of messages to send to the kafka topic",
+    default=100,
+    show_default=True,
+)
+@click.option(
+    "--kwargs",
+    type=str,
+    help="Task function keyword arguments",
+)
+@click.option(
+    "--args",
+    type=str,
+    help="Task function arguments",
+)
+@click.option(
+    "--task-function",
+    type=str,
+    help="The function name of the task to execute located in the module",
+    required=True,
+)
+@click.option(
+    "--path",
+    type=str,
+    help="The path to the task module",
+    required=True,
+)
+def taskbroker_send_tasks(
+    path: str,
+    task_function: str,
+    args: str,
+    kwargs: str,
+    repeat: int,
+) -> None:
+    import importlib
+
+    try:
+        module = importlib.import_module(path)
+        func = getattr(module, task_function)
+    except Exception as e:
+        click.echo(f"Error: {e}")
+        raise click.Abort()
+
+    for _ in range(repeat):
+        task_args = [] if not args else eval(args)
+        task_kwargs = {} if not kwargs else eval(kwargs)
```

Comment:
If the args/kwargs aren't changing per call we could do them once.

Reply:
Oops, yes